src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom'
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path='/' exact component={Home}/>
+    <Route path='/about' exact component={About}/>
```

Comment:
this might be just out of preference but put exact right before the path in your Route; this might be just out of preference but do what you prefer.